x-pack/plugins/apm/public/components/fleet_integration/apm_agents/agent_instructions_accordion.tsx

```diff
@@ -242,11 +242,11 @@ export function AgentInstructionsAccordion({
                     <EuiFlexItem grow={false}>
                       <EuiBetaBadge
                         label={i18n.translate(
-                          'xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.betaBadge.label',
-                          { defaultMessage: 'BETA' }
+                          'xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.experimentalBadge.label',
+                          { defaultMessage: 'Experimental' }
```

Comment:
```suggestion
                          { defaultMessage: 'Technical preview' }
```

Our Kibana terminology has changed to "Technical preview".

Reply:
I wonder if we should use https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/components/shared/experimental_badge.tsx with its tooltip content as well.

Reply:
Good spot, it has to be exported from the root of the plugin, I've created a `TechnicalPreviewBadge` component in APM and used that 
<img width="740" alt="image" src="https://user-images.githubusercontent.com/5831975/166665779-62882c24-b6ad-4576-8273-51e8d84d2ed2.png">